cli/commands/caddy.js

```diff
@@ -12,7 +12,9 @@ module.exports = function ({ program, run }) {
         .description('lists all domains you have configured in caddy')
         .option('--user', 'allows you to add a domain on behalf of a user (requires sudo)')
         .action(async (options) => {
-            if (options?.user) username = options.user
+            username = options?.user && !isAdmin
```

Comment:
if i'm reading this right, this (and the other instances of this code) will assign `username` to `options.user` even if `options.user` is undefined. might make more sense to rewrite this using if statements instead of ternaries to be more clear as well

Reply:
If the value is there, and you're an admin, it assigns the value to that. If the value is there, but you're not an admin, it kills the process with an error. 

But yeah I can make it an if statement